pkg/ddl/tests/resourcegroup/resource_group_test.go

```diff
@@ -137,6 +137,16 @@ func TestResourceGroupBasic(t *testing.T) {
 	}
 	g = testResourceGroupNameFromIS(t, tk.Session(), "y")
 	checkFunc(g)
+	tk.MustExec("alter resource group y RU_PER_SEC=6000")
+	checkFunc = func(groupInfo *model.ResourceGroupInfo) {
+		re.Equal(true, groupInfo.ID != 0)
+		re.Equal("y", groupInfo.Name.L)
+		re.Equal(groupID.Load(), groupInfo.ID)
+		re.Equal(uint64(6000), groupInfo.RURate)
+		re.Equal(int64(6000), groupInfo.BurstLimit)
```

Comment:
In the issue， we can see the result as follows：
```
create resource group rg3 ru_per_sec=2000;
alter resource group rg3 ru_per_sec=5000;
2. What did you expect to see? (Required)
burst_limit = 2000
```
So why here is `6000` not `4000`?

Reply:
Sry, I wrote the wrong ddl statement in the issue. It should be 
`create resource group rg3 ru_per_sec= 5000; `
`alter resource group rg3 ru_per_sec=2000;`